Formula/d/docbook.rb

```diff
@@ -1,5 +1,5 @@
 class Docbook < Formula
-  desc "Standard SGML representation system for technical documents"
```

Comment:
any reference for this change?

Reply:
I suspect these package descriptions were once copied from Debian. But in Debian, the package "docbook" is DocBook SGML and the package "docbook-xml" is DocBook XML, while in Homebrew, the package "docbook" is DocBook XML (and DocBook SGML is not packaged (except at https://github.com/petere/homebrew-sgml/blob/master/docbook-sgml.rb)). So it looks like this was simply a mixup when this was first transcribed.